### ISSUE TITLE
Changing the base coin dynamically

### ISSUE THREAD

**Issue by toluwanicareer** [3s3s/opentrade#30]
IT is not documented whether it is possible to change the base coin dynamically, how can that feature be added

**Comment by bihonglu** [comment]
It appears to me that this is not possible. But it is possible to change base coin from Marycoin to some other coin , say BTC

**Comment by vitorgamer58** [comment]
Exatamente, está exchange é feita para ter uma base Coin e ir adicionando outras, não permite ter muitos pares diversos exemplo: BTC-XMR DSH-NBR IOP-LTC

**Comment by bihonglu** [comment]
It probably better to support multiple base coins, say 3 coins, one for BTC, one for LTC, one for doge.
I know MC is the only base coin for openTrade, but for others, we may pick a different coin, Ideally 2 to 3 base coins.  Beyond 3, I see little need for that.

**Comment by CryptorClub** [comment]
My question how to run same script with different base coin on separate subdomain/page?

**Comment by bihonglu** [comment]
@CryptorClub :  I do not think it is possible for current setup.  Only one main coin as basecoin is allowed. 
I think this would be major upgrade on this platform, an important feature upgrade.

**Comment by CryptorClub** [comment]
Maybe two separate scripts, that uses same Users registration/login DB...

**Comment by bihonglu** [comment]
It could be two separate folder and separate  URL for two different basecoin.  But hard part is to share the user login and balance information so that users do not login and register in two different OpenTrade site. In other word, it can be done with two basecoin, but a lot of work will be needed.

**Comment by bihonglu** [comment]
I do believe supporting two or three main basecoin is  very important and high priority feature upgrade for OpenTrade.  I hope the dev team can look into this as top need.

**Comment by diakas** [comment]
Support this idea. A few basic coins-what you need!
